src/components/PhotoAlbum/PhotoAlbum.tsx

```diff
@@ -33,10 +33,11 @@ export const PhotoAlbum = ({ photos }: { photos: any[] }) => {
           distance={Math.abs(currentPhoto - index)}
           onClick={() => setCurrentPhoto(index)}
           active={(index === currentPhoto).toString()}
-          src={`https:${photo.fields.file.url}`}
-          height={300}
-          width={600}
+          src={`https:${photo.fields.file.url}?&w=500`}
```

Comment:
What this `&w=500` does? if it's the width of the image, i would guess it's not doing much because of the width property given below

Reply:
it's a query parameter to load the image from the CMS resized to 500px width. Without it, the image would be loaded in its original size and only resized in the browser (that's how i understand it). but if we notice that it doesn't make much difference in performance, we can remove it :)

Reply:
Got it. I think you could change it to what Lazaro pointed - with the height param also